### PR TITLE
Fix/aut 86/excape double quotes

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
https://oat-sa.atlassian.net/browse/AUT-86

### Recognised issue
When exporting using QTI metadata exporter, when double quotes were used with coma, exporter handler was using coma as a content separator. 

### Solution
Replace double quotes for csv headers source and values to single quote.

### CODE REVIEW

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by pnal — CHANGES_REQUESTED** [review]
Please try to escape with double quotes like it described in RFC:
https://www.ietf.org/rfc/rfc4180.txt
image

File: model/Export/Extractor/OntologyExtractorRunner.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ class OntologyExtractorRunner
     public static function run(core_kernel_classes_Resource $item, core_kernel_classes_Property $classProperty)
     {
         $itemProperty = $item->getProperty($classProperty->getUri());
-        $labelItemProperty = $itemProperty->getLabel();
+        $labelItemProperty = str_replace('"', '\'', $itemProperty->getLabel());
```
[the remark this answers is not in the document]
Comment (bartlomiejmarszal) [reply model/Export/Extractor/OntologyExtractorRunner.php:42]: It's valid argument. IMO we should stick with replacing quotes with single quote to keep consistency. Although It's open for discussion.

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by pnal — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-bart.playground.kitchen.it.taocloud.org:49191/

**Pull request merged**